### PR TITLE
Improve license details re recent jslibs work #403

### LINKED ISSUE #403: Improve license details re recent jslibs work
We have more recently audited our jslibs repo ([rockstor-jslibs](https://github.com/rockstor/rockstor-jslibs)) to include a full breakdown of all associated licenses and expressed them in the newly added README.md file in that repository according to [SPDX License List](https://spdx.org/licenses/) standards. This specification has also now been expressed in our in-development (testing branch) Poetry based build system via the pyproject.toml file update in:
https://github.com/rockstor/rockstor-core/pull/2449
where we summarise our respective rockstor-core and rockstor-jslibs licenses in SPDX compliant terms as:
```
# Source0: (rockstor-core) is GPL-3.0-or-later, Source1: (rockstor-jslibs) has mixed licensing:
license = "GPL-3.0-or-later AND (MIT AND Apache-2.0 AND GPL-3.0-or-later AND LGPL-3.0-or-later AND ISC)"
```
Given that when folks use Rockstor in it's fully working, installed, variant it is comprised of both rockstor-core and rockstor-jslibs we are obliged to reflect this fact by referencing all licenses involved. As such it is proposed that we update our user facing docs in areas such as:
https://rockstor.com/docs/interface/system/update_channels.html
to reflect this fact and emphasise that all licenses referenced are:
"... both FSF Free/Libre, & OSI approved according to: [SPDX License List](https://spdx.org/licenses/)"
as we do in the above referenced README.md in our rockstor-jslibs repo.

We likely have other references within our docs that should similarly be updated, depending on the actual number of references we may want to create instead a centralised reference where we can more easily express the exact breakdown of all components involved, however we do already have this in-place within our repositories. And the gist of it is that we have two repos that are, respectively, single licensed (rockstor-core) and mixed-licensed (rockstor-jslibs). Hence the above SPDX formatting regarding bracketing and the embedded use of the "AND" directive.

### PR DESCRIPTION
**Pull request by phillxnet**
### This pull request's proposal
To clarify the multiple source/repo nature and consequent compound licensing of our distributed package and improve licensing language/consistency through-out.

#### Includes
- Normalise on license/s spelling.
- Use only SPDX spec licensing references.
- Move/improve our FAQ license section for emphasis/clarity.
- Minor update of installer into text and licensing references.

Fixes #403

### Checklist
- [x] With the proposed changes no Sphinx errors or warnings are generated.
- [x] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

File: interface/system/update_channels.rst [hunk 1]
```diff
@@ -3,31 +3,50 @@
...
+  Indicating the combined works in this jslibs repository.
+
+Making the package license, as per the `Fedora Project Wiki (Packaging:LicensingGuidelines)
+<https://fedoraproject.org/wiki/Packaging:LicensingGuidelines#Mixed_Source_Licensing_Scenario>`_:
+
+* *"GPL-3.0-or-later AND (MIT AND Apache-2.0 AND GPL-3.0-or-later AND LGPL-3.0-or-later AND ISC)"*
+
+See the `SPDX License List <https://spdx.org/licenses/>`_ for details on the above assertions.
+
+As such Rockstor the package is fully Open Source, there is no contributor agreement, and
```
Comment (FroggyFlox) [inline interface/system/update_channels.rst:31]: Minor style edits:
```suggestion
As such, Rockstor the package is fully Open Source; there is no contributor agreement, and
```

**Review by FroggyFlox — APPROVED** [review]
@phillxnet , looks good to me!

File: interface/system/update_channels.rst (hunk 1 above)
Comment (phillxnet) [reply interface/system/update_channels.rst:31]: @FroggyFlox Thanks, this does read better.

**Comment by phillxnet** [comment]
@FroggyFlox Thanks for the review. Much appreciated.
I'll merge and publish to PRODUCTION shortly there after.

**Comment by phillxnet** [comment]
PR product published.

**Pull request merged**